docs/releases/1.11.txt

```diff
@@ -384,6 +384,11 @@ Serialization
   :class:`~datetime.timedelta` objects (used by
   :class:`~django.db.models.DurationField`).
 
+Settings
+~~~~~~~~
+
+* Settings values are now cached in `LazySettings.__dict__` upon access.
```

Comment:
If you believe this could be backwards-compatible it could be okay to mention it there. Otherwise, this doesn't meet the usual threshold for a minor feature.